CHANGELOG.md

```diff
@@ -8,6 +8,35 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.5.4] - 2024-05-29
```

Comment:
Why aren't none of the performance improvements for Check mentioned?

https://github.com/openfga/openfga/pull/1550
https://github.com/openfga/openfga/pull/1593
https://github.com/openfga/openfga/pull/1589

Reply:
Please advise how to articulate

Reply:
[#1568](https://github.com/openfga/openfga/pull/1568)

Reply:
## Changed

- Performance improvements in Check. Less unnecessary spans created (https://github.com/openfga/openfga/pull/1550 and https://github.com/openfga/openfga/pull/1589), less calls to database if the request was cancelled or timed out (https://github.com/openfga/openfga/pull/1593).

Reply:
You dropped #1514 😅 also please mention that is specifically for Check api

Reply:
#1514 is an issue. #1593 is already marked as a fix.